### PR TITLE
Initial PR

### PR DESCRIPTION
**Pull request by derek-heycar**
## Initial PR

- TypeScript
- eslint, prettier 
- CSS - Stylelint 
- Storybook 
- Lerna
- IDE .vscode setting 
- Build tool rollup
- Jest component testing 
- Jest screenshot testing config 
- Initial packages: CSS-vars, Theming, Button

### CODE REVIEW

**Review by mwagdi — APPROVED** [review]

**Review by mwagdi — COMMENTED** [review]
@derek-heycar I get `Failed to resolve a `react-scripts` package` while attempting to start storybook

**Review by mwagdi — APPROVED** [review]

File: packages/vars/src/typography.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+:root {
+  --font-family-system: Objektiv, sans-serif, 'Helvetica Neue', Helvetica, Arial;
+  --font-family-styrene: Objektiv, sans-serif, 'Helvetica Neue', Helvetica,
```
Comment (hey-igor) [inline packages/vars/src/typography.css:3]: what does `styrene` mean?

**Review by hey-igor — APPROVED** [review]
lgtm :) just have one question

File: packages/vars/src/typography.css (hunk 1 above)
Comment (derek-heycar) [reply packages/vars/src/typography.css:3]: Thanks for noticing that. But actually, the css-variables package has not been finished yet. I thought to use it as an intermediate variable for the font, but I've checked, and it's not needed now.
I'll replace it.

File: .storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    });
+
+    config.plugins.push(
+      new MiniCssExtractPlugin(),
+      new CssMinimizerPlugin({
+        parallel: true,
+        test: cssRegex,
+      }),
+
+      // new OptimizeCSSAssetsPlugin({
```
Comment (ggonchar) [inline .storybook/main.js:119]: Should we rather remove those lines?

**Pull request merged**